cardano-crypto-tests/src/Test/Crypto/VRF.hs

```diff
@@ -208,6 +222,84 @@ prop_naturalToBytes (NonNegative sz) n =
     sz >= 8 ==>
       bytesToNatural (naturalToBytes sz (fromIntegral n)) == fromIntegral n
 
+--
+-- Praos <-> BatchCompatPraos VerKey conversion
+--
+prop_pubKeyToBatchComopat :: VerKeyVRF PraosVRF -> Bool
+prop_pubKeyToBatchComopat vk =
+  rawSerialiseVerKeyVRF (vkToBatchCompat vk) == rawSerialiseVerKeyVRF vk
```

Comment:
Consider changing the return type to `Property`, and using `===` instead of `==`. This will produce better reporting of test failures, listing not only the name of the test and the inputs, but also the exact operands to the failed comparison.